### PR TITLE
feat: add C `ndarray` implementation for `blas/base/saxpy`

### PR DESCRIPTION
**Pull request by aman-095**
Progresses #2039.

## Description

> What is the purpose of this pull request?

This pull request:

-   adds C `ndarray` API for `blas/base/saxpy`.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   progresses #2039  

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/blas/base/saxpy/manifest.json [hunk 1]
```diff
@@ -233,7 +261,9 @@
         "-lblas"
       ],
       "libpath": [],
-      "dependencies": []
+      "dependencies": [
+        "@stdlib/strided/base/min-view-buffer-index"
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/base/saxpy/manifest.json:268]: @aman-095 I believe you are missing `@stdlib/blas/base/shared`. This also needs to be fixed in `daxpy`.

Comment (kgryte) [reply lib/node_modules/@stdlib/blas/base/saxpy/manifest.json:268]: I think this skipped notice as we don't explicitly test CBLAS.

Comment (kgryte) [reply lib/node_modules/@stdlib/blas/base/saxpy/manifest.json:268]: This applies for other configurations in this file.

Comment (aman-095) [reply lib/node_modules/@stdlib/blas/base/saxpy/manifest.json:268]: Yes, I think I missed that. Updated.

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @aman-095!

**Pull request merged**